_posts/en/posts/2021-05-01-release-0.21.1.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Bitcoin Core 0.21.1 Released With Taproot Activation Code
+name: blog-release-0.21.1
+id: en-blog-release-0.21.1
+lang: en
+type: posts
+layout: post
+
+## If this is a new post, reset this counter to 1.
+version: 1
+
+## Only true if release announcement or security annoucement.  English posts only
+announcement: 1
+
+excerpt: >
+  Bitcoin Core 0.21.1 is now available with support for activating and
+  enforcing taproot.
+---
+Bitcoin Core version 0.21.0 is now available for [download][download
```

Comment:
I think you mean 0.21.1, not 0.21.0

Reply:
Fixed in force push.  Rebuilding for updated preview and tests now, should be about three minutes until that's all done.